packages/react/src/__tests__/ReactProfiler-test.internal.js

```diff
@@ -2639,32 +2659,28 @@ describe('Profiler', () => {
         ]);
         onRender.mockClear();
 
-        expect(onInteractionScheduledWorkCompleted).toHaveBeenCalledTimes(1);
-        expect(
-          onInteractionScheduledWorkCompleted,
-        ).toHaveBeenLastNotifiedOfInteraction(highPriUpdateInteraction);
+        expect(onInteractionScheduledWorkCompleted).toHaveBeenCalledTimes(0);
 
         advanceTimeBy(500);
         jest.advanceTimersByTime(500);
         await originalPromise;
-        expect(ReactTestRenderer).toHaveYielded([
-          'Suspend [loaded]',
-          'Text [loading]',
-          'Text [updated]',
-          'Promise resolved [loaded]',
-        ]);
+        expect(ReactTestRenderer).toHaveYielded(['Promise resolved [loaded]']);
         expect(renderer).toFlushAndYield(['AsyncText [loaded]']);
         expect(renderer.toJSON()).toEqual(['loaded', 'updated']);
 
         expect(onRender).toHaveBeenCalledTimes(1);
         expect(onRender.mock.calls[0][6]).toMatchInteractions([
           initialRenderInteraction,
+          highPriUpdateInteraction,
```

Comment:
The way this test changed is a little interesting. Our "hi-pri update" causes us to re-call AsyncText which means that in a sense, the update now *is* blocked on the suspended component before it fully resolves. I can see the argument either way for how to account for this. The good news is that if we only scheduled an update on the sibling then they would stay independent.